source/layouts/topics.haml

```diff
@@ -13,8 +13,8 @@
 
   .container
     .row
-      .col-xs-3
-        = partial 'support_sidebar'
+      .col-xs-2.hidden-xs
+        / = partial 'support_sidebar'
```

Comment:
Sure thing, my reaction was more to the style than the functionality, but it wasn't a strong reaction and I'm fine leaving it as is.